mnoptical/link.py

```diff
@@ -3,6 +3,8 @@
 from pprint import pprint
 from numpy import errstate
 from mnoptical.node import LineTerminal, Roadm, Amplifier
+from mnoptical.edfa_params import fibre_spectral_attenuation
```

Comment:
All tests are failing because the fibre_spectral_attenuation list is not in edfa_params.py in this PR. 
Might be worth adding it to see if tests are passing

Reply:
Yes, that should have been included in this PR instead.